index.md

```diff
@@ -7,8 +7,8 @@ DO NOT EDIT! This is only a placeholder to define the nabvar and is deleted to r
 :maxdepth: 1
 :hidden:
 
-July 2022 event <2022-07_sprint/schedule>
-People <2022-07_sprint/sprint_parties/index>
+Mar 2023 event <2023-03_sprint/index>
```

Comment:
> We are missing "Mar 2023 event" at top left of menu. Not sure where to make the adjustment in the file.

This was updated to `2023-03_sprint/schedule` which does not exist. I noticed this thanks to an sphinx warning in the rtd logs:

    /home/docs/checkouts/readthedocs.org/user_builds/pymc-data-umbrella/checkouts/189/index.md:6: WARNING: toctree contains reference to nonexisting document '2023-03_sprint/schedule'